atlp-rwanda/e-commerce-mavericcks-bn
# bug-fx create seeders for admin and role for admin

## Describe the purpose of this pull request

The aim here is to generate dummy data for both the admin user and admin role. Additionally, I intended to enhance security by implementing hashed passwords. Although I attempted to create a reusable hashedPassword function, I encountered difficulties in its utilization, yet I managed to integrate it into the auth controller and user controller.


## Testing Instructions

To validate the changes:

1. Execute the seed to generate fresh dummy data for both the admin user and role.
2. Create an account to verify the functionality of the hash password function.

## Related Issues

Missing dummy data for admin user

## Checklist

Please review the following checklist and make sure all tasks are complete before submitting:
